Automattic/studio
# Add Assistant Markdown parsing

Adds Markdown parsing and styles to the AI Assistant screen.


https://github.com/Automattic/studio/assets/643285/85858a22-a992-4577-aad6-7cc093a168b2


Resolves Automattic/dotcom-forge#7456

## Proposed Changes

- Adds `remarkable` library for Markdown parsing
- Adds `highlight.js` library for syntax highlighting
- Adds sensible default styles for Markdown formatting and code highlighting

## Testing Instructions

1. Run `npm install` to install new dependencies
2. Start app with AI Assistant flag enabled: `STUDIO_AI=true npm start`
3. Create a site and navigate to Assistant tab
4. Ask some questions in the text input, such as "Show me some Markdown" or "How do I install WordPress?"
5. Observe server responses are formatted using styling and syntax highlighting 


## Pre-merge Checklist
